spec/classes/foreman_proxy__plugin__discovery_spec.rb

```diff
@@ -18,11 +18,12 @@
           tftproot = '/var/lib/tftpboot'
       end
 
-      it { should contain_foreman_proxy__plugin('discovery') }
-      it { should contain_foreman_proxy__feature('Discovery') }
-
       describe 'without paramaters' do
-        it { should_not contain_foreman__remote_file("#{tftproot}/boot/fdi-image-latest.tar") }
+        it { should compile.with_all_deps }
+        it { should contain_foreman_proxy__plugin('discovery') }
+        it { should contain_foreman_proxy__feature('Discovery') }
+        it { should_not contain_foreman_proxy__remote_file("#{tftproot}/boot/fdi-image-latest.tar") }
+        it { should_not contain_exec('untar fdi-image-latest.tar') }
```

Comment:
kind of meta, but I think `is_expected.to` / `is_expected.not_to` is generally considered best practice these days.

Reply:
You're right. I copied the spec from puppet-foreman and didn't replace it.

Reply:
Correction: these was an existing file. I at least wanted to keep it consistent within this file.